src/marqo/core/constants.py

```diff
@@ -12,8 +14,9 @@
 MARQO_DOC_HYBRID_TENSOR_SCORE = '_tensor_score'
 MARQO_DOC_HYBRID_LEXICAL_SCORE = '_lexical_score'
 
-MARQO_STRUCTURED_HYBRID_SEARCH_MINIMUM_VERSION = '2.10.0'
-MARQO_UNSTRUCTURED_HYBRID_SEARCH_MINIMUM_VERSION = '2.11.0'
+MARQO_STRUCTURED_HYBRID_SEARCH_MINIMUM_VERSION = semver.VersionInfo.parse('2.10.0')
```

Comment:
I actually meant to create these in the class using them as a class variable. But this is probably better. Just make sure there's no  regression by finding and reviewing all usages of these

Reply:
Sure Farshid, I've made sure to review all the usages & made corresponding changes